autoload/lsp/omni.vim

```diff
@@ -194,14 +195,21 @@ function! lsp#omni#get_vim_completion_item(item, ...) abort
                 \ 'dup': 1,
                 \ 'kind': l:kind}
 
+    " check support user_data.
+    if !s:is_user_data_support && g:lsp_text_edit_enabled
+        echohl WarningMsg
```

Comment:
We should be using echo error util function.

Reply:
Fixed 728e60b9b437767163c5a426c7a943c7e64a2c05.